README.md

```diff
@@ -0,0 +1,27 @@
+# xangle-selfie-mint
+
+## Introduction
+
+dApp built on top of Aptos.
+
+## Installation
+
+```bash
+cd mint-dapp
+npm i
+export NODE_OPTIONS=--openssl-legacy-provider
```

Comment:
why is this nescesary?

Reply:
Due to use of legacy openssl provider by node

Reply:
I think this is likely an issue with your environment, i.e using an old version of node?

Reply:
Can you check what node version you are using from your end?
I want to lock the version so that everyone can use it.